packages/blocks/src/api/parser/get-block-attributes.js

```diff
@@ -58,6 +68,9 @@ export const toBooleanAttributeMatcher = ( matcher ) =>
  */
 export function isOfType( value, type ) {
 	switch ( type ) {
+		case 'rich-text':
```

Comment:
This will eventually get documented for the usage with block `attributes`. In this PR, it would be worth documenting, the rationale behind it so we know why it introduced a direct dependency between `@wordpress/blocks` and `@wordpress/rich-text`. The "Why" section would be a great place for it.

Reply:
Yes, was focussed on getting it right and tests passing. Does the new PR description help?

Reply:
This is an excellent summary! It makes crystal clear why this refactoring has to happen 😄